.github/workflows/docker.yaml

```diff
@@ -58,3 +66,18 @@ jobs:
             docker tag image $DOCKERHUB_ID:latest
             docker push $DOCKERHUB_ID:latest
           fi
+          
+
+      - name: Run Trivy vulnerability scanner
+        uses: aquasecurity/trivy-action@2a2157eb22c08c9a1fac99263430307b8d1bc7a2
+        with:
+          image-ref: '${{ steps.github_id.outputs.result }}:${{ github.sha }}'
+          format: 'template'
+          template: '@/contrib/sarif.tpl'
+          output: 'trivy-results.sarif'
+          severity: 'CRITICAL,HIGH'
+
+      - name: Upload Trivy scan results to GitHub Security tab
+        uses: github/codeql-action/upload-sarif@v1
+        with:
+          sarif_file: 'trivy-results.sarif'
```

Comment:
We may want to scan the image before we push it, to avoid pushing known vulnerable images.

In that case, we may want to refactor further, by using the proper image name straight for building, instead of `image`.